tests/console/avocado_prepare.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Avocado setup and network bridge configuration
+#          Test system has to be registered baremetal
+# Maintainer: Jozef Pupava <jpupava@suse.cz>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils qw(systemctl zypper_call);
+use version_utils qw(is_sle is_opensuse);
+use Utils::Backends 'use_ssh_serial_console';
```

Comment:
Wouldn't  be better to use  select_serial_terminal?

Reply:
changed to select_serial_terminal, but select_serial_terminal is not setting serialdev to /dev/sshserial and without use of use Utils::Backends 'use_ssh_serial_console' in test before it would fail